add CONTRIBUTING.md (#31)
coder/modules: README.md

Check out the [Coder Registry](https://registry.coder.com) for instructions to integrate modules into your template.

# Contructing a Module

> See the [CONTRIBUTING.md](./CONTRIBUTING.md) file for instructions on how to construct and publish your own module to the Coder Registry.

Reviewer: ```suggestion
See [CONTRIBUTING.md](./CONTRIBUTING.md) for instructions on how to construct and publish a module to the [Coder Registry](https://registry.coder.com).
```
